### PR TITLE
Fetch OMN price data

### PR DESCRIPTION
**Pull request by KadenZipfel**
Closes: https://github.com/protofire/omen-exchange/issues/1853

Prices are pretty wild because it's all rinkeby data

### CODE REVIEW

**Comment by gem1no** [comment]
code looks good. circling back on this and doing proper testing over weekend/monday

File: app/src/hooks/useTokenPrice.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  const [tokenPrice, setTokenPrice] = useState(new BigNumber(0))
+
+  const { data: data, error: error, loading: loading } = useQuery<GraphResponse>(query, {
+    notifyOnNetworkStatusChange: true,
+    skip: false,
+    variables: { id: address },
+  })
+
+  useEffect(() => {
+    if (data && data.tokenPrice) {
```
Comment (hexyls) [inline app/src/hooks/useTokenPrice.tsx:33]: I think the path here might be `data.token.usdPerToken`, prob need to update the `GraphResponse` type to match graph data

File: app/src/components/market/sections/market_pooling/scalar_market_pool_liquidity.tsx [hunk 2]
```diff
@@ -571,6 +572,8 @@ export const ScalarMarketPoolLiquidity = (props: Props) => {
     }
   }
 
+  const { tokenPrice } = useTokenPrice(getToken(networkId, 'omn').address)
```
Comment (hexyls) [inline app/src/components/market/sections/market_pooling/scalar_market_pool_liquidity.tsx:575]: I think token address should be lowercase here like you did with the others, could maybe update `useTokenPrice` to make addresses lowercase automatically so we don't have to worry about it in the future

File: app/src/hooks/useTokenPrice.tsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+      setTokenPrice(bigNumberify(data.tokenPrice))
```
Comment (hexyls) [inline app/src/hooks/useTokenPrice.tsx:34]: Once the path mentioned above is updated, bigNumberify is freaking out on [this market](http://localhost:3000/#/0xb4277c368b817db02bab8b684c9bfe10200461f7) with an `invalid BigNumber string value` error message, not really sure why 🤔

**Review by hexyls — CHANGES_REQUESTED** [review]

**Review by pimato — APPROVED** [review]
works!

**Review by hexyls — APPROVED** [review]
🙌

File: app/src/hooks/useTokenPrice.tsx [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    if (data && data.token.usdPerToken) {
+      setTokenPrice(Number(data.token.usdPerToken))
+    }
+  }, [data])
+
+  console.log(tokenPrice)
+
+  return {
+    tokenPrice,
+    status: error ? Status.Error : loading ? Status.Loading : Status.Ready,
```
Comment (Mi-Lan) [inline app/src/hooks/useTokenPrice.tsx:44]: Small suggestion....When I was working on history table I found out that If its set like this it doesn't always display state correctly and when changed to this 
`status: error ? Status.Error : loading ? Status.Loading : data ? Status.Ready : Status.Loading,`
heres console logs with that implemented in this pr 
Screenshot 2021-06-01 at 14 29 31
And here is it like its now :
Screenshot 2021-06-01 at 14 29 48
Although dunno if it causes some other issues when done like this?

**Review by Mi-Lan — APPROVED** [review]
Looks good!

**Pull request merged**